Assets/Project/Scripts/GamePlayScene/ObjectUnifier.cs

```diff
@@ -12,10 +12,20 @@ public class ObjectUnifier : MonoBehaviour
         [Tooltip("画像の横縦比")]
         public float ImageRatio = 1f;
 
+        private SpriteRenderer _renderer;
+
         private void Awake()
         {
-            var originalWidth = GetComponent<SpriteRenderer>().size.x;
-            var originalHeight = GetComponent<SpriteRenderer>().size.y;
+            _renderer = GetComponent<SpriteRenderer>();
+            if (_renderer.sprite != null) {
+                Unify();
+            }
+        }
+
+        public void Unify()
+        {
+            var originalWidth = GetComponent<SpriteRenderer>().sprite.bounds.size.x;
+            var originalHeight = GetComponent<SpriteRenderer>().sprite.bounds.size.y;
```

Comment:
```suggestion
            var originalWidth = _renderer.sprite.bounds.size.x;
            var originalHeight = _renderer.sprite.bounds.size.y;
```
`Awake()`で`_renderer`取得しているならそれ使いたい。

それから、`public`メソッドとして外部から呼ぶこともあるなら
```
public void Unify(){
    if (_renderer.sprite != null) return;
}
```
こっちで例外処理した方がよさそう

Reply:
a8b05ed で修正した